### ISSUE TITLE
[UI] UI for likes

### ISSUE THREAD

**Issue by furkanmutlu** [furkanmutlu/fullstack-image-app#60]
Since now we have the like functionality, It would be nice to have a page/tab to display users likes.

Questions:
>  Should we have separate like page for each API?  If yes, what are the possible UI solution comes to your mind ?
>  Should we have a one page with all likes ? Segmented by its API source? 
> Any other ideas ?

**Comment by elyessAR** [comment]
interesting...but I didn't get the whole idea, is this going to be to display the number of likes of each gif/picture?

**Comment by furkanmutlu** [comment]
Yeah thats the question. It will be either for each page/api, with tabs like below: (just created on bootstrap editor)
<img width="841" alt="image" src="https://user-images.githubusercontent.com/45897337/209784897-b403d8b0-6734-4c1f-9820-4b51fa24d44b.png">


OR we will create a new page, with collapsable segment. And all likes will be stored in there.
